etc/view.xml

```diff
@@ -7,20 +7,20 @@
                 <height>140</height>
             </image>
             <image id="cart_cross_sell_products" type="thumbnail">
-                <width>320</width>
-                <height>320</height>
+                <width>224</width>
```

Comment:
Please verify image sizes for 767px screen width. Grid product tiles has 368px width and image is stretched and looks bad. I think we should consider using max possible width as minimum default for all thumbnails

Reply:
good catch, fixed